src/util/Constants.js

```diff
@@ -463,6 +463,23 @@ exports.Colors = {
   NOT_QUITE_BLACK: 0x23272A,
 };
 
+/**
+ * The value set for the verification levels for a guild:
+ * * None
+ * * Low
+ * * Medium
+ * * (╯°□°）╯︵ ┻━┻
+ * * ┻━┻ ﾐヽ(ಠ益ಠ)ノ彡┻━┻
+ * @typedef {string} VerificationLevels
```

Comment:
```suggestion
 * @typedef {string} VerificationLevel
```

The type should be singular here to be consistent with our other typedefs for constants.